src/content/en/updates/posts/2015/12/background-sync.markdown

```diff
@@ -40,7 +40,7 @@ If the user tries to send a message when they have zero connectivity, then, than
 
 {% ytvideo l4e_LFozK2k %}
 
-Background sync hasn’t hit the main release of Chrome yet, so if you want to try this you’ll need either [Chrome Dev for Android](https://play.google.com/store/apps/details?id=com.chrome.dev&hl=en), or [Chrome Canary for desktop](https://www.google.com/chrome/browser/canary.html). You’ll also need to enable `chrome://flags/#enable-experimental-web-platform-features` and restart the browser. Then:
+As of March 2016, Background sync is now available in Chrome from version 49 and above.  You can see it action by following:
```

Comment:
The last sentence is missing a word. While you're there, please delete the word 'now' from the first sentence.